RiotSharp/Requester.cs

```diff
@@ -1,14 +1,15 @@
 ﻿using System;
 using System.Collections.Generic;
-using System.IO;
+using System.Linq;
 using System.Net;
+using System.Net.Http;
 using System.Threading.Tasks;
 
 namespace RiotSharp
 {
     class Requester
     {
-        protected string rootDomain;
+        protected string RootDomain;
```

Comment:
its just a style thing, from the default Resharper settings.  Would you like me to change it back?